server/apps/rules/routing_rules.py

```diff
@@ -21,6 +21,15 @@
 STATE_ROUTED = 'routed'
 
 
+WEEKDAYS = ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN']
```

Comment:
Thanks for the update, @sivakuna-aap, I was not aware of that.

(FWIW, this should have been reflected in the `superdesk/server`'s README...)